test/controllers/OpenPaymentsControllerSpec.scala

```diff
@@ -560,5 +560,14 @@ class OpenPaymentsControllerSpec extends ControllerBaseSpec {
         }
       }
     }
+    "the user is insolvent and not continuing to trade" should {
+
+      "return 403 (Forbidden)" in new Test {
+
+        override val authResult: Future[~[Enrolments, Option[AffinityGroup]]] = successfulAuthResult
```

Comment:
No need to override `authResult` as it's already set to `successfulAuthResult` by default in the trait